### DISCUSSION TITLE
Which linux version of the system does the MOOSE support?

### DISCUSSION THREAD

**Discussion by KangChenRui** [idaholab/moose#22527] [Q&A General] [answered]
hello!
Which linux version of the system does the MOOSE support?

I am currently reinstalling the system.

**Comment by GiudGiud** [comment, marked as answer]
this page lists the requirements
https://mooseframework.inl.gov/getting_started/installation/manual_installation_gcc.html
we dont keep track of that as closely as we generally always support linux
